src/table/TableHeadWrapper.jsx

```diff
@@ -46,12 +51,18 @@ export default function TableHeadWrapper({
       <TableRow className="sn-table-row">
         {tableData.columns.map((column, columnIndex) => {
           const tabIndex = columnIndex === 0 ? '0' : '-1';
+          const currentSortDir = SORT_NOTATIONS[column.sortDirection];
+          const ariaLabelText = `${column.label}. ${currentSortDir}. Press space to sort on this column.`;
+          const isTableSortActive = layout.qHyperCube.qEffectiveInterColumnSortOrder[0] === columnIndex;
+
           return (
             <TableCell
               key={column.id}
               align={column.align}
               className={`${classes.head} sn-table-head-cell sn-table-cell`}
               tabIndex={tabIndex}
+              aria-label={ariaLabelText}
+              aria-sort={column.sortDirection}
```

Comment:
What does the screen reader say here? The direction is now in three places. `aria-label`, `aria-sort` and the hidden label. AFAIK it is better to have a hidden label, that's why Li implemented that. Also, what does it say when the sort isn't active on a column? Is it sorted on that column, but not primarily?

Reply:
For each column there is a default sorting order that comes from `tableData` i guess, and screen reader reads through the `aria-label` not `aria-sort`. 
when sort is not active on a column, it will tell user the default order of the column that comes from `tableData`.

btw, this is the default behaviour of the previous table which i have implemented here.

Reply:
So does aria-sort do anything? And you don't hear the hidden label anymore?
I think we need to include if it is actively sorted or not. There is no indication on which column it is sorted on primarily.

Reply:
I just included the `aria-sort` in order to match with the older version since you mentioned that the previous table is the complete version of this one. but seems like you are right, it does not do any thing so far if the screen reader is going to read the `aria-label` only.

One suggestion that i can make here is that we can add the current (active) sort to the `aria-label` attributes for columns. by doing this way, user will always here a sentence like this: 
`columnName. Sorted ascending. **currently table sorted on ColumnX in ascending way.** Press space key to sort on this column.`

Reply:
Ah ok. Sorry that I was unclear: I guess we want the old behaviour more or less, not the implementation necessarily.

I think that aria-label overrides the other announcements, that's why it is not best practice to use in some cases. So if we keep aria-sort, then we could announce the instructions using the hidden label. I had a look here: https://adrianroselli.com/2021/04/sortable-table-columns.html#Announce Seems like some screen readers announce when you update aria-sort, some does not (including NVDA which we support). So we definitely need to test this with NVDA.

About the active column, I misunderstood that a bit. There is only one column which is sorted. The other ones should have `aria-sort="none"`, which is default (so actually no need to add aria-sort in those cases at all). I don't think we need to say which column that is sorted when you are on a non-sorted column.